modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_builder.scss

```diff
@@ -10,4 +10,8 @@
 			font-weight: 600;
 		}
 	}
+}
+
+.read-only-container-root {
```

Comment:
I'll either do a generic helper class or be more specific. And add a nice transition.